src/io/flutter/run/bazel/BazelFields.java

```diff
@@ -139,19 +129,9 @@ public String getAdditionalArgs() {
     return additionalArgs;
   }
 
-
   @Nullable
-  public String getBazelTarget() {
```

Comment:
nit:  make this just `getTarget()` with a comment that it is either a bazel or dart target.